### PR TITLE
[TD-10546]<feature>:nodejs support json tag

### PR DESCRIPTION
**Pull request by xleili**

### CODE REVIEW

File: src/connector/nodejs/nodetaos/cinterface.js [hunk 1]
```diff
@@ -13,8 +13,15 @@ const errors = require('./error');
 const _ = require('lodash')
 const TaosObjects = require('./taosobjects');
 
-module.exports = CTaosInterface;
 
+
```
Comment (zitsen) [inline src/connector/nodejs/nodetaos/cinterface.js]: Unnecessary multiple empty lines. Use single empty line between code blocks.

File: src/connector/nodejs/nodetaos/cinterface.js [hunk 2]
```diff
@@ -219,8 +245,8 @@ var TaosField = Struct({
   'name': char_arr,
 });
 TaosField.fields.name.type.size = 65;
-TaosField.defineProperty('type', ref.types.char);
-TaosField.defineProperty('bytes', ref.types.short);
+TaosField.defineProperty('type', ref.types.uint8);
+TaosField.defineProperty('bytes', ref.types.uint16);
```
Comment (zitsen) [inline src/connector/nodejs/nodetaos/cinterface.js]: `bytes` type is int16. See C struct:

```c
typedef struct taosField {
  char     name[65];
  uint8_t  type;
  int16_t  bytes;
} TAOS_FIELD;
```

**Review by zitsen — CHANGES_REQUESTED** [review]
Please update.

File: src/connector/nodejs/nodetaos/cinterface.js (hunk 1 above)
Comment (xleili) [reply src/connector/nodejs/nodetaos/cinterface.js]: done

File: src/connector/nodejs/nodetaos/cinterface.js (hunk 2 above)
Comment (xleili) [reply src/connector/nodejs/nodetaos/cinterface.js]: done

**Review by zitsen — APPROVED** [review]
LGTM.

**Review by sangshuduo — APPROVED** [review]
LGTM

**Pull request merged**